### ISSUE TITLE
GSE45878  GTEX Affymetrix

### ISSUE THREAD

**Issue by ANiknejad** [BgeeDB/expression-annotations#43]
Normality.

GSM1117974
tissue: Bone Marrow
tissue_detail: Cells - Leukemia cell line (CML)

mais ça on ne prend pas, ce n'est pas une condition normale pour 2 raisons:
1.- ce sont des cell lines
2.- ces cell lines sont anormales (leukemia)

**Comment by ANiknejad** [comment]
I am sorry for Angélique's annotation....

**Comment by fbastian** [comment]
I was assigned to this issue, but I don't remember what I am supposed to do :p It seems that you already clarified that we can't use this.

**Comment by julien-roux** [comment]
The annotation might not all be lost since the same samples are probably used fro RNA-seq too, right?

**Comment by fbastian** [comment]
Hmm, I think we didn't capture the sample ID, that would allow to make the link. I guess it is easy to retrieve, but I think it is safer to re-annotate the RNA-Seq data, as they might have been updated with new information.

**Comment by julien-roux** [comment]
Yes that's how I understood sample IDs, isn't it like this?
Maybe they don't have SRS IDs for affy chips because these cannot be deposited in SRA?

**Comment by fbastian** [comment]
@julien-roux: Well, I never found any SRS ID reused for different data.

**Comment by ANiknejad** [comment]
There is no other GTEx Affy than GSE45878

Platform:
    GPL16977 
837 Samples

**Comment by ANiknejad** [comment]
@mimiblis 

please comments all the line (affymetrixChip.tsv and microarrayExperiment.tsv)

**Comment by mimiblis** [comment]
done: f23473c

**Comment by ANiknejad** [comment]
We need to check if GPL16977 mapping to Ensembl is available on Biomart
http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL16977

[HuGene-1_1-st] Affymetrix Human Gene 1.1 ST Array [HuGene11stv1_Hs_ENSG version 14.1.0]
This is a "custom-commercial" chip apparently, so will probably not be in Ensembl

Ensembl has mapping to  Affymetrix Microarray hugene 1 0 st v1 probeset ID(s), which may have a lot of probesets in common: should we use this?
Or should we use an external mapping file (as we used to do with the Su et al. dataset)?
@fbastian what is your opinion?

**Comment by fbastian** [comment]
Throwing the potato to @julien-roux: do they provide mappings to Entrez/Ensembl genes in their custom CDF files? (http://brainarray.mbni.med.umich.edu/Brainarray/Database/CustomCDF/CDF_download.asp according to GEO)

**Comment by julien-roux** [comment]
I don't think this is the official page for this array, but indeed there seem to be a mapping file here:
http://mbni.org/customcdf/20.0.0/ensg.download/hugene11st_Hs_ENSG_20.0.0.zip

Actually if you look at he probeset names at: http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL16977 (scroll down a bit), you can see that the mapping is fairly simple, just remov ethe "_at" at the end of probeset names ;)

**Comment by fbastian** [comment]
So, we should be able to provide a mapping file suitable for our pipeline?

**Comment by julien-roux** [comment]
Sure! 
By the way isn't GTEx providing the mapping for this chip?

**Comment by julien-roux** [comment]
Please see this email of @smoretti to Ensembl mailing list:
http://lists.ensembl.org/pipermail/dev/2015-September/011430.html
We might maybe use the  Affymetrix Human Gene 1.0 ST Array mapping instead

**Comment by fbastian** [comment]
It will maybe be available in Ensembl 83?

**Comment by julien-roux** [comment]
You mean Ensembl 84? Nope, nothing new on that side...
